### ISSUE TITLE
Instalation issue with python 3.9 and last version of conda

### ISSUE THREAD

**Issue by enriiquee** [nanoporetech/tombo#425]
Is there any way to update the package for the new versions of python and conda?

**Comment by adalisan** [comment]
Yes, actually, bumping up the version to include  changes since 1.5.1, and putting up on  pypi would help a lot.
